styles/pages/landing/_howFuctions.scss

```
@@ -10,18 +10,18 @@
   .font{
     font-size: 40px;
     font-family: $font-family;
-    font-weight: 600;
+    font-weight: 50px;
```

el font-weight, puede recibir valores de 100 a 600 sin unidad de medidas
```suggestion
    font-weight: 300;
```

---

Gracias Fer, ya hice el cambio.